src/pages/Loans/LoansOverview/components/LoansMarkets/SupplyMarket.tsx

```diff
@@ -0,0 +1,131 @@
+import { CurrencyIdLiteral } from '@interlay/interbtc-api';
+import { Key, useState } from 'react';
+
+import { ReactComponent as ChevronRight } from '@/assets/img/icons/chevron-right.svg';
+import { CoinIcon, Span, Stack } from '@/component-library';
+import { SupplyAssetData, SupplyPositionData } from '@/utils/hooks/api/loans/use-get-loans-data';
+
+import { SupplyModal } from '../SupplyModal';
+import { StyledApyTag, StyledAsset, StyledTableWrapper } from './LoansMarkets.style';
+import { MarketTable } from './MarketTable';
+import { SupplyAssetsColumns, SupplyAssetsTableRow, SupplyPositionColumns, SupplyPositionTableRow } from './types';
+
+// TODO: translations
+const supplyAssetsColumns = [
+  { name: 'Asset', uid: SupplyAssetsColumns.ASSET },
```

Comment:
why uid instead of id?

Reply:
Which documentation? Happy to leave it as it is if it's necessary, but otherwise better to use `id` to match our existing data structures.